lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -7,6 +7,12 @@ const { domSymbolTree } = require("../helpers/internal-constants");
 const createHTMLCollection = require("../../living/html-collection").create;
 const notImplemented = require("../../browser/not-implemented");
 const { reflectURLAttribute } = require("../../utils");
+const { constructTheFormDataSet } = require("../xhr/FormData-impl");
```

Comment:
Ideally let's move this to `../xhr/helpers/formdata.js` or similar (i.e. -impl classes should not generally contain helper utilities)